.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["es2015"]
+  "presets": ["es2015", "es2017"]
```

Comment:
I know we discussed it before, but as you're changing this anyway, should we use the `env` preset now?

Reply:
... maaaaaybe 😸